idea_town/frontend/static-src/test/lib/router.js

```diff
@@ -0,0 +1,100 @@
+import test from 'tape';
+import Router from '../../app/lib/router';
+import app from 'ampersand-app';
+import Me from '../../app/models/me';
+import Experiments from '../../app/collections/experiments';
+
+test(`🖳  Running Tests for ${__filename} ␄`, a => a.end());
+
+test('landing redirects correctly if logged in with addon', t => {
+  t.plan(2);
+
+  app.me = new Me({
+    user: {
+      id: 'gary@busey.biz',
+        addon: {url: '/lol nothing matters'}
+      }
+  });
```

Comment:
Looks like quite a bit of duplicated test setup in this file. Can you extract this to a setup method?